Report_Generation/Report_graphs.py

```diff
@@ -1,12 +1,12 @@
 import pandas as pd
 import psycopg2
-from io import BytesIO
 import matplotlib.pyplot as plt 
 import sys
+from matplotlib.backends.backend_pdf import PdfPages
+
 
 class queryandTransform:
```

Comment:
I don't think it's a good idea to combine two different functions together. Data query and data transformation should be separated so that if one failed for whatever reason, it does not cause the other to fail. Easier to debug and write unit tests as well.